### ISSUE TITLE
[Ch 26][Notebook][Minor] §6 이 "Ch 24 와 동일 hyperparams" 라고 단언하는데 학습률만 다름 (3e-4 vs 5e-4) — 같은 노트북의 변경점 표와 자기모순

### ISSUE THREAD

**Issue by yum-yeom** [yoon-gu/neuqes-101#225]
> **요약** — Ch 26 §6 의 제목·도입문이 **"Ch 24 와 동일 hyperparams"·"hyperparams 모두 동일"** 이라고 단언하는데, 같은 노트북의 변경점 표와 실제 코드는 **학습률만 다르다**(Ch 24 `3e-4` → Ch 26 `5e-4`)고 적고 있습니다. 한 노트북 안의 자기모순입니다.

## 무엇이 문제인가

기준: master `d6c78e1`

| 위치 | 서술 |
|---|---|
| 노트북 `ma27` (§6 제목) | `## 6. Trainer 로 사전학습 (**Ch 24 와 동일 hyperparams**)` |
| 노트북 `ma27` (도입문) | Ch 24 와 *완전히 같은* Trainer 패턴 — 모델 클래스·collator·**hyperparams 모두 동일** |
| 노트북 `ma27` (바로 아래 불릿) | `max_steps=1500`, `batch_size=32`, **`lr=5e-4`** |
| 노트북 `ma2` (변경점 표) | 학습률 \| 3e-4 (scratch) \| **5e-4** ← 한국어 vocab 에 맞춘 미세 조정 |
| 코드 `c28` | `learning_rate=5e-4,   # scratch 사전학습 표준` |
| Ch 24 코드 `c27` | `learning_rate=3e-4,` |

나머지(`max_steps=1500`, `batch_size=32`, `weight_decay=0.1`, `betas=(0.9,0.95)`, `warmup_steps=100`, `cosine`, `max_grad_norm=1.0`, `eval_steps=150`, `fp16`)는 실제로 Ch 24 와 동일합니다 — **어긋나는 건 학습률 한 항목뿐**입니다.

덧붙여 `c28` 의 주석 `# scratch 사전학습 표준` 도 어색합니다. Ch 24 역시 scratch 사전학습인데 `3e-4` 를 쓰므로, 이 값이 "표준"이라서 5e-4 인 것이 아니라 `ma2` 가 적은 대로 *한국어 vocab 에 맞춘 조정* 입니다.

## 왜 고쳐야 하나

이 챕터의 설계 원칙이 **"변경점 한 가지 (언어 축)"** 이고, 그래서 §6 이 "hyperparams 모두 동일" 을 강조합니다. 그런데 실제로는 학습률도 함께 바뀌었으므로, 두 챕터의 loss 를 비교할 때(#221 관련 §6 관전 포인트, "영어 약 3.7 vs 한국어 약 4.5") **언어 차이인지 학습률 차이인지 통제되지 않습니다.** 학습자가 "변수 하나만 바꿔 비교한다"는 이 책의 방법론을 잘못 배우게 되는 자리입니다.

## 제안

- §6 제목: `Trainer 로 사전학습 (Ch 24 와 같은 셋업, 학습률만 5e-4)`
- 도입문: "모델 클래스·collator·hyperparams 모두 동일" → **"모델 클래스·collator 는 동일하고, hyperparams 는 학습률만 `3e-4 → 5e-4` 로 다릅니다 (변경점 표 참조)"**
- `c28` 주석: `# scratch 사전학습 표준` → **`# 한국어 vocab(4,000)에 맞춘 조정 — Ch 24 는 3e-4`**
- §6 관전 포인트에서 Ch 24 와 도달 loss 를 비교하는 문장에 *학습률도 다르다* 는 단서를 한 줄 추가

## 재현

```bash
git checkout d6c78e1
python3 - <<'PY'
import json
for f,l in [("26_ko_tiny_gpt/26_ko_tiny_gpt.ipynb","Ch26"),("24_gpt_tinystories/24_gpt_tinystories.ipynb","Ch24")]:
    nb=json.load(open(f))
    for c in nb["cells"]:
        s="".join(c["source"])
        if "learning_rate" in s and c["cell_type"]=="code":
            print(l, [x.strip() for x in s.split("\n") if "learning_rate" in x])
PY
```

**전파 경로**: 노트북 `ma27`·`c28` → `executed/` → `pages/26-ko_tiny_gpt-practice.md` → `book/chapters/ch26_ko_tiny_gpt.tex` → EPUB.

**심각도**: Minor — 코드는 정상 동작하고 수치도 맞습니다. 다만 *변경점 한 가지 원칙* 을 스스로 어긴 서술이라 동결(9/20) 전 정리 권장.

Related to #50 (챕터 이슈), #223 ("Ch 24 와 동일" 주장이 어긋나는 다른 자리)
